pkg/datasources/secrets.go

```diff
@@ -107,7 +107,7 @@ func ReadSecrets(ctx context.Context, d *schema.ResourceData, meta any) diag.Dia
 	req := sdk.NewShowSecretRequest()
 
 	handleLike(d, &req.Like)
-	err := handleExtendedIn(d, &req.In)
+	err := handleExtendedIn(d, &req.ExtendedIn)
```

Comment:
I'm open for discussions, but I'm not sure if it's necessary. On the other hand, in Snowflake, is just `IN`, and initially `ExtendedIn` was used only to distinguish from `In` with limited filtering capabilities.

Reply:
For me, it's more readable that way because I know it's the extended version, but both options are ok.

Reply:
I'll leave it as is, meaning `In` will not be changed to `ExtendedIn`